packages/vitepress-plugin-git-changelog/src/types/index.ts

```diff
@@ -66,6 +66,10 @@ export interface Commit {
    * The repository URL.
    */
   repo_url?: string
+  /**
+   * The digits of commit hash.
+   */
+  commitHashDigits?: number
```

Comment:
Should we move this configuration to UI as part of the Vue component options since this is a style related configuration instead of data fetching related?

Reply:
You can do by your own,  and the code is just a reference.

Reply:
What do you mean by "You can do by your own, and the code is just a reference"?

Reply:
I mean you can do this change (move the configuration), cause I have no idea how to

Reply:
I will migrate them to UI in next PR when this PR got merged.